core/shared/src/main/scala/fs2/concurrent/pubsub/Bounded.scala

```diff
@@ -0,0 +1,78 @@
+package fs2.concurrent.pubsub
+import fs2.Chunk
+import scala.collection.immutable.{Queue => ScalaQueue}
+
+object Bounded {
+
+  /** unbounded fifo strategy **/
+  def fifo[A](maxSize: Int): PubSubStrategy[A, Chunk[A], ScalaQueue[A], Int] =
+    strategy(maxSize)(UnboundedQueue.fifo[A])(_.size)
+
+  /** unbounded lifo strategy **/
+  def lifo[A](maxSize: Int): PubSubStrategy[A, Chunk[A], ScalaQueue[A], Int] =
+    strategy(maxSize)(UnboundedQueue.lifo[A])(_.size)
+
+  /**
+    * Creates bounded strategy, that won't accepts elements if size produced by `f` is >= `maxSize`
+    *
+    * @param maxSize    Maximum size of enqueued `A` before this is full
+    * @param szOf       Function to extract current size of `S`
+    */
+  def strategy[A, S](maxSize: Int)(strategy: PubSubStrategy[A, Chunk[A], S, Int])(
+      f: S => Int): PubSubStrategy[A, Chunk[A], S, Int] =
```

Comment:
`f` is not named as `@param` says, `szOf`.

Reply:
done thanks